hw/bsp/dialog_da1469x-dk-pro/syscfg.yml

```diff
@@ -38,6 +38,14 @@ syscfg.defs:
         description: 'Specifies path to ED25519 private key PEM file'
         value: ''
 
+    USER_AES_SLOT:
+        description: 'Specifies AES key slot for OTP user data encryption and decryption'
+        value: -1
+
+    USER_AES_KEYSIZE:
```

Comment:
Do we really want this configurable?  The user data keys are all AES-256.  So there isn't any choice in which we can use on Mist.

Reply:
removed